packages/manager/modules/exchange/src/account/add/account-add.controller.js

```diff
@@ -57,6 +57,16 @@ export default class ExchangeAccountAddController {
     return this.fetchingAccountCreationOptions();
   }
 
+  $doCheck() {
```

Comment:
i think we have to avoid to use the `$` char to distinguish Angular snippet code from own.
may be it's an angular function ?

Reply:
Ah i just googled bit and yep it's an angular function.

Reply:
Yep this function is part of the component life cycle hooks